front/app/containers/Admin/settings/index.tsx

```diff
@@ -1,105 +1,94 @@
+      label: formatMessage(messages.tabAreas),
+      url: '/admin/settings/areas',
+    },
+    {
+      name: 'policies',
+      label: formatMessage(messages.tabPolicies),
+      url: '/admin/settings/policies',
+    },
+  ]);
 
-interface State {
-  tabs: ITab[];
-}
-
-class SettingsPage extends React.PureComponent<
-  Props & WrappedComponentProps & WithRouterProps,
-  State
-> {
-  constructor(props: Props & WrappedComponentProps & WithRouterProps) {
-    super(props);
-    const { formatMessage } = this.props.intl;
-
-    this.state = {
-      tabs: [
-        {
-          name: 'general',
-          label: formatMessage(messages.tabSettings),
-          url: '/admin/settings/general',
-        },
-        {
-          name: 'customize',
-          label: formatMessage(messages.tabCustomize),
-          url: '/admin/settings/customize',
-        },
-        {
-          name: 'registration',
-          label: formatMessage(messages.tabRegistration),
-          url: '/admin/settings/registration',
-        },
-        {
-          label: formatMessage(messages.tabTopics),
-          name: 'topics',
-          url: '/admin/settings/topics',
-        },
-        {
-          name: 'areas',
-          label: formatMessage(messages.tabAreas),
-          url: '/admin/settings/areas',
-        },
-        {
-          name: 'policies',
-          label: formatMessage(messages.tabPolicies),
-          url: '/admin/settings/policies',
-        },
-      ],
-    };
-  }
-
-  handleData = (insertTabOptions: InsertConfigurationOptions<ITab>) => {
-    this.setState(({ tabs }) => ({
-      tabs: insertConfiguration(insertTabOptions)(tabs),
-    }));
+  const handleData = (insertTabOptions: InsertConfigurationOptions<ITab>) => {
+    setTabs(insertConfiguration(insertTabOptions)(tabs));
   };
 
-  render() {
-    const { formatMessage } = this.props.intl;
-
-    const resource = {
-      title: formatMessage(messages.pageTitle),
-    };
+  return (
+    <>
+      <Outlet id="app.containers.Admin.settings.tabs" onData={handleData} />
+      <NavigationTabs>
+        {tabs.map(({ url, label }) => (
+          <Tab key={url} active={matchPathToUrl(url).test(pathname)}>
+            <Link to={url}>{label}</Link>
+          </Tab>
+        ))}
+      </NavigationTabs>
 
-    return (
-      <>
-        <Outlet
-          id="app.containers.Admin.settings.tabs"
-          onData={this.handleData}
+      <TabsPageLayout>
+        <HelmetIntl
+          title={messages.helmetTitle}
+          description={messages.helmetDescription}
         />
-        <TabbedResource resource={resource} tabs={this.state.tabs}>
-          <HelmetIntl
-            title={messages.helmetTitle}
-            description={messages.helmetDescription}
-          />
-          <div id="e2e-settings-container">
-            <RouterOutlet />
-          </div>
-        </TabbedResource>
-      </>
-    );
-  }
-}
+        <Box
+          id="e2e-initiatives-admin-container"
```

Comment:
Should it be `e2e-settings-container`? Otherwise `front/cypress/e2e/admin/admin.cy.ts` may fail.

Reply:
Indeed, nice catch 💪